iron-input.html

```diff
@@ -68,18 +68,20 @@
       },
 
       /**
-       * Set to true to prevent the user from entering invalid input. The new input characters are
-       * matched with `allowedPattern` if it is set, otherwise it will use the `type` attribute (only
-       * supported for `type=number`).
+       * Set to true to prevent the user from entering invalid input. If `allowedPattern` is set, 
+       * The new input characters are matched with that. If more than one character is typed or pasted in,
```

Comment:
the "if more than one character is pasted in the input string will be ignored entirely" is a lie right now, so I think we should omit it. Unless you meant is ignored with regards to being prevented, in which case that's not super clear :(

How about "Set to true to prevent the user from entering invalid input. If `allowedPattern` is set, any character typed by the user will be matched against that pattern, and rejected if it's not a match. Note that right now, any pasted input will not be checked against the pattern, and will be automatically accepted. If `allowedPattern` is not set, the `type` attribute will be used for `type=number`"